server/controllers/auth.js

```diff
@@ -115,3 +119,73 @@ exports.logoutUser = asyncHandler(async (req, res, next) => {
 
   res.send("You have successfully logged out");
 });
+
+// @route POST /auth/forgot
+// @desc Send reset password link to selected email
+// @access Public
+exports.sendResetLink = asyncHandler(async (req, res, next) => {
+  try {
+    const { email } = req.body;
+    const user = await User.findOne({ email });
+
+    if (!email) {
+      res.status(400).send({ error: "Email is required" });
+    }
+
+    if (!validator.isEmail(email)) {
+      res.status(400).send({ error: "Invalid email" });
+    }
```

Comment:
It will be good to combine this one with the validation above ==> ` if (!email || !validator.isEmail(email)) res.status(400).send({ error: "Missing or invalid email" });
`

Reply:
Totally, thanks.